### PR TITLE
utils: Add extension properties to convert values to mebi- and gibibytes

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Add convenience properties for converting values to mebibytes and gibibytes.

### CODE REVIEW

File: utils/common/src/main/kotlin/Extensions.kt [hunk 1]
```diff
@@ -226,6 +226,16 @@ fun File.toSafeUri(): URI {
...
 
+/**
+ * Return the bytes equal to this [Int] number of mebibytes (MiB).
+ */
+inline val Int.mebibytes get(): Long = this * 1024L * 1024L
+
+/**
+ * Return the bytes equal to this [Int] number of gibibytes (GiB).
+ */
+inline val Int.gibibytes get(): Long = this * 1024L * 1024L * 1204L
```
Comment (sschuberth) [inline utils/common/src/main/kotlin/Extensions.kt:237]: Should the implementation call `mebibytes` internally?

**Review by sschuberth — COMMENTED** [review]
BTW, should we ever require more of this, we could take a look at https://github.com/nacular/measured.

File: utils/common/src/main/kotlin/Extensions.kt (hunk 1 above)
Comment (mnonnenmacher) [reply utils/common/src/main/kotlin/Extensions.kt:237]: I wasn't sure if that's not overdoing it, what would you prefer?

Comment (sschuberth) [reply utils/common/src/main/kotlin/Extensions.kt:237]: I think I'd have a slight tendency towards doing it, as it IMO increases readability (by reducing the operations) and it documents the relation between the units.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**